plugin/src/py/android_screenshot_tests/device_name_calculator.py

```diff
@@ -73,6 +76,10 @@ def _architecture_text(self):
         architecture = self._execute_adb_command(['shell', 'getprop', 'ro.product.cpu.abi'])
         return architecture.rstrip()
 
+    def _language_text(self):
+        architecture = self._execute_adb_command(['shell', 'getprop', 'ro.product.locale'])
```

Comment:
Does this work properly with pseudo locales?

Reply:
I didn't know about pseudo locales. I test it and 'ro.product.locale' doesn't return pseudo locales, but 'persist.sys.locale' does.
Here is the output with pseudo locale:
```
$ adb shell getprop | grep locale
[persist.sys.locale]: [en-XA]
[ro.product.locale]: [en-US]
```

Here is the output with english locale:
```
$ adb shell getprop | grep locale
[persist.sys.locale]: [en-US]
[ro.product.locale]: [en-US]
```

I'm going to change 'ro.product.locale' to 'persist.sys.locale', are you okay with this? @xiphirx

Reply:
Hmm... Is there information on the availability of `persist.sys.locale` ? Ideally I think we'd want to check that it exists + use it if there is a value, but otherwise fall back to `ro.product.locale`